README.md

```diff
@@ -10,7 +10,7 @@ The repo contains Scraper and Data from Investment firms, in use for the Open Pe
 
 ## Installation
 
-Not yet.
+pip install -r requirements.txt.
```

Comment:
Describe what they do in this step, something like:
```markdown
- Install dependencies `pip install -r requirements.txt`.
```

And the command should be in `code` tags (backtick, this: `).